File: calc.js
// Simple Calculator
function calculate() {
    const num1 = parseFloat(document.getElementById('num1').value);
    const num2 = parseFloat(document.getElementById('num2').value);
    const operator = document.getElementById('operator').value;
    let result;

    switch (operator) {
        case '+':
            result = num1 + num2;
            break;
        case '-':
            result = num1 - num2;
            break;
        case '*':
            result = num1 * num2;
            break;
        case '/':
            result = num2 !== 0 ? num1 / num2 : 'Error: Division by zero';
            break;
        default:
            result = 'Invalid operator';
    }

    document.getElementById('result').textContent = `Result: ${result}`;
}

// HTML structure
document.write(`
    <div style="font-family: Arial; text-align: center; margin-top: 50px;">
        <h1>Simple Calculator</h1>
        <input type="number" id="num1" placeholder="Enter first number" style="margin: 5px; padding: 10px; font-size: 16px;">
        <select id="operator" style="margin: 5px; padding: 10px; font-size: 16px;">
            <option value="+">+</option>
            <option value="-">-</option>
            <option value="*">*</option>
            <option value="/">/</option>
        </select>
        <input type="number" id="num2" placeholder="Enter second number" style="margin: 5px; padding: 10px; font-size: 16px;">
        <button onclick="calculate()" style="margin: 5px; padding: 10px; font-size: 16px; cursor: pointer;">Calculate</button>
        <p id="result" style="font-size: 18px; margin-top: 20px;"></p>
    </div>
`);

File: calc.c
#include <stdio.h>

void calculator() {
    double num1, num2, result;
    char operator;

    printf("\nSimple Calculator\n");
    printf("Enter first number: ");
    scanf("%lf", &num1);

    printf("Enter an operator (+, -, *, /): ");
    scanf(" %c", &operator);

    printf("Enter second number: ");
    scanf("%lf", &num2);

    switch (operator) {
        case '+':
            result = num1 + num2;
            break;
        case '-':
            result = num1 - num2;
            break;
        case '*':
            result = num1 * num2;
            break;
        case '/':
            if (num2 != 0) {
                result = num1 / num2;
            } else {
                printf("Error: Division by zero is not allowed.\n");
                return;
            }
            break;
        default:
            printf("Invalid operator. Please use +, -, *, or /.\n");
            return;
    }

    printf("Result: %.2lf\n", result);
}

int main() {
    calculator();
    return 0;
}
